exp/services/recoverysigner/internal/account/account.go

```diff
@@ -12,3 +11,9 @@ type Identities struct {
 	PhoneNumber string
 	Email       string
 }
+
+// Present indicates if the Identities contains at least one identity. Returns
+// false if it is an empty/zero value.
+func (i Identities) Present() bool {
```

Comment:
Nit: I personally like leading bool methods with `Is`, just to make things clearer:
```suggestion
func (i Identities) IsPresent() bool {
```

Reply:
I've liked this pattern too in other languages in the past, but in Go code it's more common to omit the `Is`. Grabbing a random example the [`Response`](https://godoc.org/net/http#Response) type has `Close` and `Uncompressed`, but there are loads of examples of this in the stdlib and other packages.